etc/templates/generic.slurm.template

```diff
@@ -2,11 +2,13 @@
 #SBATCH -p $QUEUE
 #SBATCH -N 1
 #SBATCH -t $WALL_CLOCK
+#SBATCH -A $ACCOUNT
+#SBATCH -q $QOS
 #SBATCH --ntasks-per-node 1
 #SBATCH --cpus-per-task $CPUS
 #SBATCH -J glide_$USER_NAME
 
-MB=$(( 4000*$CPUS ))
+MB=$(( 4096*$CPUS ))
```

Comment:
Seems a little uncomfortable to have this magic number in two places (here and ctrl_execute).

Reply:
Yes, this number of "MB" is going to tell HTCondor how much memory to advertise, so I do need a way to pass through the Memory for the Slurm job submission into the HTCondor configuration. Think, think ...

Reply:
ok I think  I can set the value in one place for now via 
self.commandLineDefaults["MEMPERCORE"] = 4096
and then that value will go to the Slurm config and the htcondor config. 
I'll test that out and update;

Reply:
Committed updates to derive from a single location.